.github/workflows/haskell.yml

```diff
@@ -102,13 +102,18 @@ jobs:
             runner: ubuntu-latest
             cabal_version: 3.2
 
-          - name: 9.2.1
-            ghc_version: 9.2.1
+          - name: 9.2.5
+            ghc_version: 9.2.5
             runner: ubuntu-latest
             cabal_version: 3.6
 
-          - name: 9.4.2
-            ghc_version: 9.4.2
+          - name: 9.4.4
+            ghc_version: 9.4.4
+            runner: ubuntu-latest
+            cabal_version: 3.8.1.0
+
+          - name: 9.6.0
+            ghc_version: 9.6.0
```

Comment:
We have a working ghc head version CI in streamly. You can try adding it like this: https://github.com/composewell/streamly/blob/ab28951742e0a423a27762a5c9cccaf49745c253/.github/workflows/haskell.yml#L193

Reply:
Done